beacon_node/network/src/sync/backfill_sync/mod.rs

```diff
@@ -898,6 +942,49 @@ impl<T: BeaconChainTypes> BackFillSync<T> {
         Ok(())
     }
 
+    /// Retries partial column requests within the batch by creating new requests for the failed columns.
+    pub fn retry_partial_batch(
+        &mut self,
+        network: &mut SyncNetworkContext<T>,
+        batch_id: BatchId,
+        id: Id,
+        failed_columns: HashSet<ColumnIndex>,
+        mut failed_peers: HashSet<PeerId>,
+    ) -> Result<(), BackFillError> {
```

Comment:
Doesn't look like we need to return a `Result` here, it's always `Ok(())`

Reply:
meant to return an error here if the batch doesn't exist. Should never happen and it shouldn't fail silently if it does